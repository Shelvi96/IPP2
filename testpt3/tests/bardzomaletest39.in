NEW ID

74;2:9:6>1;85
6>4998689
4>96;:772
1073695>520058124;
50:41>7;425:8
:;>7890;10816
:59357;:4>1816:9;8
33948090>:63
0009;>03
13802932:>60260;61:
861062>:2748:48
2;;4:>4039
922::>3:7
158>1
;:070>642
166::8>0621:40
594229501>49793
2:100>4576:52:
7;:91;2607>26;581315
@91:7;791150;65:1;28:4
@9:63588
@2:605507
@14202827;47
@1133868822:00;:6;9845;477:
@:220743:09;035:851:3
@7084;4;:268918453790694;;4812
@590213:41131242:155
@63;22
@6;;8399332:
@:;610892
@:00064;0:::11:4:;;51973019
