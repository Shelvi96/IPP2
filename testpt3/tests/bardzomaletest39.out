216130949
0
0
0
118205557922304
21435767
204259204089741312
295424
0
0
0
11211000000000
